pkg/config/validation/validation.go

```diff
@@ -1611,16 +1611,45 @@ func ValidateProxyConfig(config *meshconfig.ProxyConfig) (errs error) {
 		errs = multierror.Append(errs, multierror.Prefix(err, "invalid proxy admin port:"))
 	}
 
-	switch config.ControlPlaneAuthPolicy {
-	case meshconfig.AuthenticationPolicy_NONE, meshconfig.AuthenticationPolicy_MUTUAL_TLS:
-	default:
-		errs = multierror.Append(errs,
-			fmt.Errorf("unrecognized control plane auth policy %q", config.ControlPlaneAuthPolicy))
+	if err := ValidateControlPlaneAuthPolicy(config.ControlPlaneAuthPolicy); err != nil {
+		errs = multierror.Append(errs, multierror.Prefix(err, fmt.Sprintf("invalid authentication polic:")))
+	}
+
+	if err := ValidateStatNameLength(config.StatNameLength); err != nil {
+		errs = multierror.Append(errs, multierror.Prefix(err, fmt.Sprintf("invalid name:")))
+	}
+
+	if err := ValidatePort(int(config.StatusPort)); err != nil {
+		errs = multierror.Append(errs, multierror.Prefix(err, fmt.Sprintf("invalid status port:")))
 	}
 
 	return
 }
 
+// The default value is set to 189 characters.
+// Envoy's internal metrics take up 67 characters, for a total of 256 character name per metric.
+// Increase the value of this field if you find that the metrics from Envoys are truncated.
+func ValidateStatNameLength(len int32) error {
```

Comment:
Envoy has already removed `StatNameLength`. This option actually does not take any effect, so no need to add validation for it.

Reply:
> Envoy has already removed `StatNameLength`. This option actually does not take any effect, so no need to add validation for it.

ok,remove it